mote/main.py

```diff
@@ -215,5 +218,17 @@ def page_not_found(error):
     return render_template("e404page.html"), 404
 
 
+@main.get("/teams")
+def teamspage():
+    path = os.path.abspath(main.config["TEAMS_FILE_LOCATION"])
+    data = dict()
+    with open(path, "r") as stream:
+        try:
+            data = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            logging.error("exc")
```

Comment:
If your goal was to log the exception, the correct syntax should be:
```suggestion
            logging.error(exc)
```